src/backend/p3/p3api.js

```diff
@@ -0,0 +1,50 @@
+const fs=require("fs/promises");
+const express = require("express");
+//const cors = require("cors"); //if encoutering errors whne using from the front end, remove comment status //didn't require it so far
+
+const app= express()
+app.use(express.json());
+//app.use(express.cors());  //if encoutering errors whne using from the front end, remove comment status s //didn't require it so far
+
+if (require.main === module) {
+  app.listen(3000, () => console.log('p3 server is running'));
+}
+
+function calculateQuote(car_value, risk_rating) {
```

Comment:
I like the function, it does what you need it to do, well done. What I might suggest however is making two functions. Declare your quote variable globally as 0 or since it is js, undefined. Then have one function for calculating yearlyPrem and one for monthlyPrem. One function, one job. Since the variables are declared globally, both of your functions can access them too.